File: 백준/Bronze/24077. 比較 （Comparison）/比較 （Comparison）.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n,m=map(int,input().split())
a=[*map(int,input().split())]
b=[*map(int,input().split())]
cnt=0
for i in range(n):
    for j in range(m):
        if a[i]<=b[j]:
            cnt+=1
print(cnt)

File: 백준/Bronze/24077. 比較 （Comparison）/比較 （Comparison）.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n, m, cnt = 0;
    cin >> n >> m;
    vector<int> a(n), b(m);
    for (int i = 0; i < n; i++)
        cin >> a[i];
    for (int i = 0; i < m; i++)
        cin >> b[i];
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            if (a[i] <= b[j])
                cnt += 1;
        }
    }
    cout << cnt;
    return 0;
}